card/mod/02_basic_types/set/all/base.rb

```diff
@@ -6,9 +6,20 @@ def show view, args
   end
 
   # NAME VIEWS
-
   simple_args = { closed: true, perms: :none }
-  view(:name,     simple_args) { card.name                           }
+  view :name, simple_args do |args|
+    return card.name unless args[:variant]
+    args[:variant].split(/[\s,]+/).inject(card.name) do |name, variant|
+      if ::Set.new([
+        :downcase, :upcase, :singularize, :pluralize, :capitalize,
+        :swapcase, :reverse, :succ
```

Comment:
these are all verbs. `variant: singularize` seems a little odd to me.

I would suggest: lowercase, uppercase, singular, plural, and capitalized.  (the others are pretty funny.  maybe they should be easter eggy?)